diff --git a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheet/BottomSheet.js b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheet/BottomSheet.js
index 8b10eb1..9c3ab2c 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheet/BottomSheet.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheet/BottomSheet.js
@@ -110,6 +110,7 @@ const BottomSheetComponent = /*#__PURE__*/(0, _react.forwardRef)(function Bottom
     backgroundComponent,
     footerComponent,
     children
+    wrapperContainerStyle
   } = props; //#endregion
   //#region layout variables
 
@@ -1110,6 +1111,7 @@ const BottomSheetComponent = /*#__PURE__*/(0, _react.forwardRef)(function Bottom
     topInset: topInset,
     bottomInset: bottomInset,
     detached: detached
+    wrapperContainerStyle: wrapperContainerStyle,
   }, /*#__PURE__*/_react.default.createElement(_reactNativeReanimated.default.View, {
     style: containerStyle
   }, /*#__PURE__*/_react.default.createElement(_bottomSheetBackgroundContainer.default, {
diff --git a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetContainer/BottomSheetContainer.js b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetContainer/BottomSheetContainer.js
index 1e5389e..f5484c7 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetContainer/BottomSheetContainer.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/commonjs/components/bottomSheetContainer/BottomSheetContainer.js
@@ -26,7 +26,8 @@ function BottomSheetContainerComponent({
   bottomInset = 0,
   shouldCalculateHeight = true,
   detached,
-  children
+  children,
+  wrapperContainerStyle
 }) {
   const containerRef = (0, _react.useRef)(null); //#region styles
 
@@ -34,7 +35,7 @@ function BottomSheetContainerComponent({
     top: topInset,
     bottom: bottomInset,
     overflow: detached ? 'visible' : 'hidden'
-  }], [detached, topInset, bottomInset]); //#endregion
+  }, wrapperContainerStyle || {}], [detached, topInset, bottomInset, wrapperContainerStyle]); //#endregion
   //#region callbacks
 
   const handleContainerLayout = (0, _react.useCallback)(function handleContainerLayout({
diff --git a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheet/BottomSheet.js b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheet/BottomSheet.js
index 62c348b..aab1cca 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheet/BottomSheet.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheet/BottomSheet.js
@@ -78,6 +78,7 @@ const BottomSheetComponent = /*#__PURE__*/forwardRef(function BottomSheet(props,
     backgroundComponent,
     footerComponent,
     children
+    wrapperContainerStyle
   } = props; //#endregion
   //#region layout variables
 
@@ -1078,6 +1079,7 @@ const BottomSheetComponent = /*#__PURE__*/forwardRef(function BottomSheet(props,
     topInset: topInset,
     bottomInset: bottomInset,
     detached: detached
+    wrapperContainerStyle: wrapperContainerStyle
   }, /*#__PURE__*/React.createElement(Animated.View, {
     style: containerStyle
   }, /*#__PURE__*/React.createElement(BottomSheetBackgroundContainer, {
diff --git a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetContainer/BottomSheetContainer.js b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetContainer/BottomSheetContainer.js
index 17b13e0..190d07d 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetContainer/BottomSheetContainer.js
+++ b/node_modules/@gorhom/bottom-sheet/lib/module/components/bottomSheetContainer/BottomSheetContainer.js
@@ -11,7 +11,8 @@ function BottomSheetContainerComponent({
   bottomInset = 0,
   shouldCalculateHeight = true,
   detached,
-  children
+  children,
+  wrapperContainerStyle
 }) {
   const containerRef = useRef(null); //#region styles
 
@@ -19,8 +20,8 @@ function BottomSheetContainerComponent({
     top: topInset,
     bottom: bottomInset,
     overflow: detached ? 'visible' : 'hidden'
-  }], [detached, topInset, bottomInset]); //#endregion
-  //#region callbacks
+  }, wrapperContainerStyle || {}], [detached, topInset, bottomInset, wrapperContainerStyle]); //#endregion
+  // #region callbacks
 
   const handleContainerLayout = useCallback(function handleContainerLayout({
     nativeEvent: {
diff --git a/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts b/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts
index 2fe2f4b..38c682b 100644
--- a/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts
+++ b/node_modules/@gorhom/bottom-sheet/lib/typescript/components/bottomSheet/types.d.ts
@@ -180,6 +180,7 @@ export interface BottomSheetProps
       >
     >
   >;
+  wrapperContainerStyle?: StyleProp<ViewStyle>;
   /**
    * View style to be applied to the background component.
    *
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
index 0f5ac23..0461239 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/BottomSheet.tsx
@@ -152,6 +152,8 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
       backgroundComponent,
       footerComponent,
       children,
+      // @see https://github.com/gorhom/react-native-bottom-sheet/pull/743
+      wrapperContainerStyle
     } = props;
     //#endregion
 
@@ -1567,6 +1569,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
               topInset={topInset}
               bottomInset={bottomInset}
               detached={detached}
+              containerProps={wrapperContainerStyle}
             >
               <Animated.View style={containerStyle}>
                 <BottomSheetBackgroundContainer
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts
index 2fe2f4b..38c682b 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheet/types.d.ts
@@ -180,6 +180,7 @@ export interface BottomSheetProps
       >
     >
   >;
+  wrapperContainerStyle?: StyleProp<ViewStyle>;
   /**
    * View style to be applied to the background component.
    *
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
index 09a4b6c..5fe1829 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/BottomSheetContainer.tsx
@@ -13,6 +13,7 @@ function BottomSheetContainerComponent({
   shouldCalculateHeight = true,
   detached,
   children,
+  containerProps: containerStyleProps
 }: BottomSheetContainerProps) {
   const containerRef = useRef<View>(null);
   //#region styles
@@ -24,8 +25,9 @@ function BottomSheetContainerComponent({
         bottom: bottomInset,
         overflow: detached ? 'visible' : 'hidden',
       },
+      containerStyleProps || {}
     ],
-    [detached, topInset, bottomInset]
+    [detached, topInset, bottomInset, containerStyleProps]
   );
   //#endregion
 
diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/types.d.ts b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/types.d.ts
index 721a918..fdfacbb 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/types.d.ts
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetContainer/types.d.ts
@@ -1,5 +1,5 @@
 import type { ReactNode } from 'react';
-import type { Insets } from 'react-native';
+import type { Insets, ViewStyle, StyleProp } from 'react-native';
 import type Animated from 'react-native-reanimated';
 import type { BottomSheetProps } from '../bottomSheet/types';
 
@@ -11,4 +11,5 @@ export interface BottomSheetContainerProps
   containerOffset: Animated.SharedValue<Insets>;
   shouldCalculateHeight?: boolean;
   children: ReactNode;
+  containerProps?: StyleProp<ViewStyle>;
 }
